Project.html

```diff
@@ -0,0 +1,131 @@
+        <nav>
+            <!-- <ul>
+                <li>About us</li>
+                <li>Career</li>
+                <li>Departments</li>
+            </ul> -->
+            <a href="aboutus">About us</a>
+            <a href="career">Career</a>
+            <a href="departmens">Departmens</a>
+        </nav>
+    </div>
+
+    <main>
+        <h2>Making the healthcare experience more human</h2>
+        <div class="paragraphs">
+
+            <div class="p1">We want patients to find the perfect doctor and book an appointment in the most easy way.
+             The patient journey should be enjoyable, and that's why we are always next to them:
+              to help them find the best possible care. Anytime, anywhere.</div>
+            <div class="p2">We also help doctors to better manage their practice and build their online reputation.
+             With our integrated end-to-end solution, doctors are able not only to improve their online presence,
+              but also to devote their time to what really matters: their patients</div>
+            
+        </div>
+    </main>
+
+    <div class="description">
+        <span class="card-1">
+            <img src="https://www.docplanner.com/icons/icon-patients.svg">
+            <h3>For Patients</h3>
+            <p>Find a doctor, book a visit and solve any health-related doubt</p>
+            <select class="drop-menu">
+                <option value="choosecountry">Choose Country</option>
+                <option value="Argentina">Argentina</option>
+                <option value="Brazil">Brazil</option>
+                <option value="Egypt">Egypt</option>
+                <option value="Qatar">Qatary</option>
+                <option value="uae">UAE</option>
+                <option value="usa">USA</option>
+                <option value="saudiarabia">Saudi Arabia</option>
+            </select>
+        </span>
+
+        <span class="card-2">
+            <img src="https://www.docplanner.com/icons/icon-doctors.svg">
+            <h3>For Doctors</h3>
+            <p>Save time managing visits and cut no-shows by half</p>
+            <select class="drop-menu">
+                <option value="choosecountry">Choose Country</option>
+                <option value="Argentina">Argentina</option>
+                <option value="Brazil">Brazil</option>
+                <option value="Egypt">Egypt</option>
+                <option value="Qatar">Qatary</option>
+                <option value="uae">UAE</option>
+                <option value="usa">USA</option>
+                <option value="saudiarabia">Saudi Arabia</option>
+            </select>
+        </span>
+
+        <span class="card-3">
+            <img src="https://www.docplanner.com/icons/icon-clinics.svg">
+            <h3>For Clinics</h3>
+            <p>Deliver an exceptional patient experience in your clinic</p>
+        </span>
+    </div>
+
+    <div class="info">
+        <span class="fact">
```

Comment:
If you want to use a generic container for a block, then use a `div` not a `span`.